packages/core/package.json

```diff
@@ -25,28 +25,45 @@
       ]
     }
   },
+  "exports": {
+    ".": [
+      {
+        "node": {
+          "import": "./node.mjs",
+          "require": "./dist/index.js"
+        },
+        "default": "./lib/index.js"
+      },
+      "./lib/index.js"
```

Comment:
fwiw, this would only be triggered by versions of node that understand "exports" but not the object form - so it kind of seems like this should either be "whatever's in main", or, you could just omit the array and use the object directly.

Reply:
Not sure why understanding exports would imply something knows cjs and not es6

Reply:
Because there are versions of node that understand exports but not conditions, and require doesn’t understand ESM.

Reply:
Isn't that why the engines mark excludes those versions of node?

Reply:
right - so then what's the array fallback for, since those are the only versions of node that need it?

Reply:
non-node tools? perhaps it should just be removed.

Reply:
That's what I'm suggesting :-) I don't know of any non-node tools that would bother supporting "exports" but not support condition objects, and it seems like a strange thing to "just in case" support.

Reply:
https://github.com/coinbase/rest-hooks/pull/992